packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -27,81 +27,69 @@
             class="p-[16px]"
             data-cy="release-highlights"
           >
-            <h1 class="font-medium text-center mb-[28px] tracking-tighter text-[22px] leading-snug text-gray-1000">
+            <h1 class="font-medium text-center mb-[8px] tracking-tighter text-[22px] leading-snug text-gray-1000">
               {{ t('majorVersionWelcome.title') }}
             </h1>
+
+            <div
+              v-if="slots['video']"
+              class="border-transparent rounded mb-6 bg-gray-50/50 border-4px text-center max-w-80vw w-688px overflow-hidden"
+            >
+              <div
+                class="bg-white rounded-t border-4px-gray-500 w-full p-24px"
+                data-cy="video-container"
+              >
+                <slot name="video" />
+              </div>
+            </div>
+
             <div class="mb-[16px]">
               <ExternalLink
-                href="https://on.cypress.io/changelog#12-0-0"
+                href="https://on.cypress.io/changelog?utm_source=splash-page&utm_campaign=v13#13-0-0"
```

Comment:
Generally we use only 2 values for `utm_source` - `Binary: App` or `Binary: Launchpad`. Is this intentionally different from that? I realize it's not documented well. There's a [sheet](https://docs.google.com/spreadsheets/d/1Cs933TLXcS63PmMlXA4wKLJtQSFH90dProvzI9oyEB4/edit#gid=0) that wrangles these values, it would be good to add these there. Medium seems more suitable than "source" for "splash page" based on the existing pattern.

But also curious - is there a particular person expecting to do some analysis later based on these params, or are we more playing it safe?

Reply:
I'm probably not the best person to ask about the `utm_source`, but it was in the approved copy for the splash page. I wonder if it makes sense to update the `utm_source` to `Binary: App` and leverage `medium` for the `splash page` . Heres the [copy](https://docs.google.com/document/d/1LjVmCnM1KTM9Uu6TqZzagCD60KRQs1gaWb3kzhJQv08/edit) for reference. I can ask the GTM team in slack

Reply:
Good idea to clarify in Slack. Being consistent is nice but there's not a hard and fast rule.